Library/Homebrew/extend/pathname.rb

```diff
@@ -29,7 +29,7 @@ def compute_disk_usage
     if self.directory?
       @file_count = 0
       @disk_usage = 0
-      self.find do |f|
+      find do |f|
```

Comment:
Will let Mike comment on the `self.exist?`. He wrote the commit a couple years back: https://github.com/Homebrew/homebrew/commit/f4d29a04943a390e39f7ce51a7f2d45a595592ca